Sources/ObservableStore/ObservableStore.swift

```diff
@@ -111,15 +111,30 @@ public struct Update<Model: ModelProtocol> {
 
     public init(
         state: Model,
-        fx: Fx<Model.Action> = Empty(completeImmediately: true)
-            .eraseToAnyPublisher(),
-        transaction: Transaction? = nil
+        fx: Fx<Model.Action>,
+        transaction: Transaction?
     ) {
         self.state = state
         self.fx = fx
         self.transaction = transaction
     }
 
+    public init(state: Model, animation: Animation? = nil) {
```

Comment:
Creating initializers for the more common case of passing in an animation directly.